<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lin_guiderClass</class>
 <widget class="QMainWindow" name="lin_guiderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="videoFrame">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>645</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSetup">
    <property name="title">
     <string>&amp;Setup</string>
    </property>
    <addaction name="actionSetupGuider"/>
    <addaction name="actionSetupDriver"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuProcessing">
    <property name="title">
     <string>&amp;Processing</string>
    </property>
    <addaction name="actionRecord"/>
    <addaction name="separator"/>
    <addaction name="actionCalibration"/>
    <addaction name="actionGuiding"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSetup"/>
   <addaction name="menuProcessing"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar_Main">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSetupGuider"/>
   <addaction name="actionSetupDriver"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionRecord"/>
   <addaction name="separator"/>
   <addaction name="actionCalibration"/>
   <addaction name="actionGuiding"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QToolBar" name="toolBar_Helper">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Toggle_Calibration_Guider"/>
   <addaction name="actionAdjust2fitCamera"/>
  </widget>
  <action name="actionSetupDriver">
   <property name="icon">
    <iconset resource="../rc/lin_guider.qrc">
     <normaloff>:/new/prefix1/driver.png</normaloff>:/new/prefix1/driver.png</iconset>
   </property>
   <property name="text">
    <string>Pulse Dev&amp;ice Settings</string>
   </property>
   <property name="toolTip">
    <string>Pulse Device Settings</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSetupGuider">
   <property name="icon">
    <iconset resource="../rc/lin_guider.qrc">
     <normaloff>:/new/prefix1/video.png</normaloff>:/new/prefix1/video.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Video Settings</string>
   </property>
   <property name="iconText">
    <string>Video Settings</string>
   </property>
   <property name="toolTip">
    <string>Video Settings</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCalibration">
   <property name="icon">
    <iconset resource="../rc/lin_guider.qrc">
     <normaloff>:/new/prefix1/calibration.png</normaloff>:/new/prefix1/calibration.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Calibration</string>
   </property>
   <property name="toolTip">
    <string>Calibration</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionGuiding">
   <property name="icon">
    <iconset resource="../rc/lin_guider.qrc">
     <normaloff>:/new/prefix1/guiding.png</normaloff>:/new/prefix1/guiding.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Guiding</string>
   </property>
   <property name="toolTip">
    <string>Guiding</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../rc/lin_guider.qrc">
     <normaloff>:/new/prefix1/about.png</normaloff>:/new/prefix1/about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About Lin-guider</string>
   </property>
   <property name="iconText">
    <string>About Lin-guider</string>
   </property>
   <property name="toolTip">
    <string>About Lin-guider</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRecord">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../rc/lin_guider.qrc">
     <normaloff>:/new/prefix1/record.png</normaloff>:/new/prefix1/record.png</iconset>
   </property>
   <property name="text">
    <string>Movie &amp;Record</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSettings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../rc/lin_guider.qrc">
     <normaloff>:/new/prefix1/settings.png</normaloff>:/new/prefix1/settings.png</iconset>
   </property>
   <property name="text">
    <string>General &amp;Settings</string>
   </property>
   <property name="toolTip">
    <string>General Settings</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Toggle_Calibration_Guider">
   <property name="icon">
    <iconset resource="../rc/lin_guider.qrc">
     <normaloff>:/new/prefix1/calibr2guider.png</normaloff>:/new/prefix1/calibr2guider.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Toggle Calibration&lt;-&gt;Guider</string>
   </property>
  </action>
  <action name="actionAdjust2fitCamera">
   <property name="icon">
    <iconset resource="../rc/lin_guider.qrc">
     <normaloff>:/new/prefix1/adjust2fit.png</normaloff>:/new/prefix1/adjust2fit.png</iconset>
   </property>
   <property name="text">
    <string>Ad&amp;just to fit</string>
   </property>
   <property name="toolTip">
    <string>Adjust to fit camera view</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../rc/lin_guider.qrc"/>
 </resources>
 <connections/>
</ui>
